routes/mapRouters.js

```diff
@@ -0,0 +1,21 @@
+const e = require("express");
+const express = require("express");
+
+const router = express.Router();
+const dev = require("../public/devs.json");
+
+//var name = "Dalia Ramos-Hernandez";
+router.get("/map/get", (req, res) => {
+  res.render("map");
+});
```

Comment:
This is a great start, but it is different from what I think /map/get should do.